src/mesh/Draco_Mesh.cc

```diff
@@ -69,7 +61,7 @@ Draco_Mesh::Draco_Mesh(
     : dimension(dimension_), geometry(geometry_),
       num_cells(safe_convert_from_size_t(num_faces_per_cell_.size())),
       num_nodes(safe_convert_from_size_t(global_node_number_.size())),
-      side_set_flag(side_set_flag_), ghost_cell_number(ghost_cell_number_),
+      side_set_flag(std::move(side_set_flag_)), ghost_cell_number(ghost_cell_number_),
```

Comment:
Same `std::move` question here - not sure why this particular vector would be picked for move semantics, while `num_faces_per_cell` below would not be... apologies if missing something!

Reply:
I had the same question, but didn't follow the thread as my goal was to allow `clang-tidy` to process all of Draco w/o warnings.  Maybe LLVM can reason about the number of copies being made and how that memory is used in the class.

Reply:
Oh, the `side_set_flag_` argument is not getting passed to any of the private function called in the constructor, which means it can be moved without corrupting object construction.